homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -99,8 +99,11 @@ def setup(hass, config):
 
     async def async_handle_update(service):
         """Handle service to update leaf data from Nissan servers."""
-        # It would be better if this was changed to use nickname, or
-        # an entity name rather than a vin.
+        _LOGGER.warning(
+            "The 'nissan_leaf.start_charge' service is deprecated and has been"
```

Comment:
I think you inverted the warnings.
```suggestion
            "The 'nissan_leaf.update' service is deprecated and has been"
```